packages/core/admin/server/register.js

```diff
@@ -17,5 +17,10 @@ module.exports = ({ strapi }) => {
     registerAdminPanelRoute({ strapi });
   }
 
-  registerDataTransfer(strapi);
+  if (
+    process.env.STRAPI_EXPERIMENTAL === 'true' &&
+    process.env.STRAPI_DISABLE_REMOTE_TRANSFER !== 'true'
```

Comment:
```suggestion
    process.env.STRAPI_DISABLE_REMOTE_DATA_TRANSFER !== 'true'
```

What do you think? My take here is that we might have different kinds of "transfer" in the future. I just wanted to make it clearer that we're talking about the data one.